src/server/auth/arboristClient.js

```diff
@@ -12,11 +12,16 @@ class ArboristClient {
     // Make request to arborist for list of resources with access
     const resourcesEndpoint = `${this.baseEndpoint}/auth/mapping`;
     log.debug('[ArboristClient] listAuthorizedResources jwt: ', jwt);
+    
+    // The POST endpoint checks for the presence of a token and validates it if present. 
+    // For retrocompatibility if the token is not present will use the clientID or username passed in the body. 
+    // We want to avoid using the body values since that could expose 
+    // other clients or usernames that are not the logged in user ones.
     const headers = (jwt) ? { Authorization: `bearer ${jwt}` } : {};
     return fetch(
       resourcesEndpoint,
       {
-        method: 'GET',
+        method: 'POST',
```

Comment:
This is a breaking change since the POST endpoint in older Arborist versions does not accept a token. I think we should add a retry here to make it backwards compatible - if the POST request doesn't work, try a GET request.

And a comment explaining the situation - we can remove the retry in a little while, when it will likely not cause issues